### ISSUE TITLE
The moodle/moodle virtual package is too restrictive

### ISSUE THREAD

**Issue by mackensen** [CLAMP-IT/moodlegist#1]
I'm in a scenario where some packages aren't blessed for Moodle 3.1, though they probably still work. Because `moodle/moodle` is a dependency you can't omit it. If you set it 3.1.0 the older packages are excluded. If you set it to 3.0.0 you can't include packages with an explicit 3.1.0 dependency. If you try something like ^3.0.0 you get warned about conflicting versions.

I think the safest solution (for now) is to drop the moodle/moodle virtual package from the build generation while retaining version search.

**Comment by juho-jaakkola** [comment]
I'm also in favor of dropping the `moodle/moodle` metapackage.

I'm working on a way to install both plugins *and* Moodle core as dependencies of a parent project: https://github.com/juho-jaakkola/moodle-project. Moodle Packagist is naturally incompatible with that approach, because Composer only processes the metapackage and does not actually install Moodle.

**Comment by mackensen** [comment]
Yeah, this feature probably isn't coming back. I disabled it on moodlegist.clamp-it.org a while ago.
